app/Admin/src/Http/Controllers/Auth/AuthController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Admin\Http\Controllers\Auth;
+
+use App\Admin\Http\Requests\Auth\LoginRequest;
+use App\Admin\Http\Requests\Auth\LogoutRequest;
+use App\Admin\Http\Resources\Auth\AdminResource;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Validation\ValidationException;
+use Knuckles\Scribe\Attributes\Authenticated;
+use Knuckles\Scribe\Attributes\BodyParam;
+use Knuckles\Scribe\Attributes\Endpoint;
+
+class AuthController
+{
+    /**
+     * @throws ValidationException
+     */
+    #[Endpoint('Admin/Login', <<<'DESC'
+    This endpoint enable users with admin role to sign in
+ DESC)]
+    #[BodyParam('email', 'password', required: true)]
+    public function login(LoginRequest $request): AdminResource
+    {
+        $credentials = $request->only('email', 'password');
+
+        if (Auth::attempt($credentials)) {
```

Comment:
Even though you did change the validation for the email specifically, we would still need to validate the role of the user to be administrator specifically, because the marketing team may have a learnhub.mk address which would allow them to login without a problem and that's not supposed to happen.